### DISCUSSION TITLE
扩展倍速变黑了

### DISCUSSION THREAD

**Discussion by sky-279** [the1812/Bilibili-Evolved#3396] [提问 / Q&A]
![批注 2022-06-03 220530](https://user-images.githubusercontent.com/60536042/171870627-5e402e15-7d6a-41fb-8d24-6215caf4c851.png)

**Comment by sky-279** [comment]
v2.2.1
Cent Browser
版本 4.3.9.248（正式版本） （32 位） (Chromium 86.0.4240.198)

**Reply by LonelySteve** [reply]
请到 issues 区提问

**Reply by sky-279** [reply]
已转issues 本条Q&A可删
